SwanProjects/bin/swan_env

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Can you add a comment about what this script does?

Reply:
I did it